LibreHardwareMonitorLib/Hardware/Psu/CoolerMaster/CoolerMasterPsu.cs

```diff
@@ -0,0 +1,248 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) 2020 Wilken Gottwalt<wilken.gottwalt@posteo.net>
+// Copyright (C) 2023 Jannis234
+// Copyright (C) LibreHardwareMonitor and Contributors.
+// Partial Copyright (C) Michael Möller <mmoeller@openhardwaremonitor.org> and Contributors.
+// All Rights Reserved.
+
+// Implemented after the Linux kernel driver corsair_psu by Wilken Gottwalt and contributers
+// Implemented after the Linux kernel driver cm_psu by Jannis234 and contributers https://github.com/Jannis234/cm-psu
+
+
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using HidSharp;
+using HidSharp.Utility;
+using LibreHardwareMonitor.Hardware.Motherboard.Lpc;
+using LibreHardwareMonitor.Hardware.Psu.Corsair;
+using static LibreHardwareMonitor.Interop.Ftd2xx;
+
+namespace LibreHardwareMonitor.Hardware.Psu.CoolerMaster;
+
+
```

Comment:
Format doc / follow code style.

Reply:
Is the code style documented anywhere? I couldn't find it.

If not, I'll study some other code and try to work it out.

Thanks :)